### PR TITLE
Update Hedgehog-Markets support

### PR DESCRIPTION
**Pull request by Juici**
Adds support for additional markets to the existing Hedgehog-Markets support.
- https://github.com/DefiLlama/DefiLlama-Adapters/pull/11714
- https://github.com/DefiLlama/DefiLlama-Adapters/pull/11719

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/hedgehog-markets: 
        
 
  
    
    
    Error: 429 Too Many Requests: {"jsonrpc":"2.0","error":{"code":-32429,"message":"max usage reached"}}
    Truncated error stack:
        at ClientBrowser.callServer (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@solana/web3.js/lib/index.cjs.js:4944:18)

**Comment by llamatester** [comment]
The adapter at projects/hedgehog-markets exports TVL: 
        
 
 
    solana                    78.09 k
    
    total                    78.09 k

**Comment by llamatester** [comment]
The adapter at projects/hedgehog-markets exports TVL: 
        
 
 
    solana                    78.01 k
    
    total                    78.01 k

File: tsconfig.json
Comment (g1nt0ki) [inline tsconfig.json:1]: can you please revert this

File: projects/hedgehog-markets/index.js
Comment (g1nt0ki) [inline projects/hedgehog-markets/index.js:1]: sorry, but this is a bit of nightmare to review, can you revert changes to this file to the version in main branch, and include only new code that you want to add?


I did some refactoring/optimization so the file is easy for me to understand, dont want to repeat that.

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
Error while running adapter at projects/hedgehog-markets: 
        
 
  
    
    
    Error: Failed to post https://mainnet.helius-rpc.com/?api-key=0109717a-77b4-498a-bc3c-a0b31aa1b3bf
    Truncated error stack:
    
        at Object.post (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/http.js:47:11)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:269:29
        at async runInChunks (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:392:22)
        at async sumTokens2 (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:216:20)
        at async tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/hedgehog-markets/index.js:20:3)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:173:9
        at async Promise.all (index 0)
        at async Promise.all (index 1)

File: tsconfig.json
Comment (Juici) [reply tsconfig.json:1]: Done in [`cc655f7`](https://github.com/DefiLlama/DefiLlama-Adapters/pull/12315/commits/cc655f7b7f1b2d1154812a1c91d6caa9236ea1e1).

File: projects/hedgehog-markets/index.js
Comment (g1nt0ki) [reply projects/hedgehog-markets/index.js:1]: @Juici thanks for indulging me.

**Pull request merged**